ee/codegen/src/generators/nodes/bases/base.ts

```diff
@@ -383,28 +383,36 @@ export abstract class BaseNode<
                 attribute: ["wrap"],
                 modulePath: adornment.base.module,
               }),
-              arguments_: filteredAttributes.map((attr) =>
-                python.methodArgument({
+              arguments_: filteredAttributes.map((attr) => {
+                const nodeConfig = NODE_DEFAULT_ATTRIBUTES[adornment.base.name];
+                const attrConfig = nodeConfig?.[attr.name];
```

Comment:
[oos] yea perfect. only thing left I could see is that we pass this `attrConfig` directly, so that `WorkflowValueDescriptor` eventually constructs `python.reference` internally. I like the idea of only raw data types going down, AstNodes building up.

but I'm looking to cut another release, so let's merge this in first